uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_description/urdf/sensors.macro.xacro

```diff
@@ -0,0 +1,113 @@
+        <identical_blocks>
+          <geometry>
+            <box size="${sensor_params['dim']['length']} ${sensor_params['dim']['width']} ${sensor_params['dim']['height']}"/>
+          </geometry>
+          <material name="blue">
+            <color rgba="0 0 0.8 1.0"/>
+          </material>
+        </identical_blocks>
+      </xacro:identical_visual_collision>
+      <xacro:default_inertial/>
+    </link>
+
+    <joint name="${sensor_params['parent_link']}_to_${name}" type="fixed">
+      <parent link="${sensor_params['parent_link']}"/>
+      <child link="${name}"/>
+      <origin xyz="${sensor_params['offset']['x']}     ${sensor_params['offset']['y']}     ${sensor_params['offset']['z']} " 
+              rpy="${sensor_params['offset']['r_rot']} ${sensor_params['offset']['p_rot']} ${sensor_params['offset']['y_rot']} "/>
+    </joint>
+
+  </xacro:macro>
+
+
+  <xacro:macro name="default_depth_camera_frame" params="camera_link_name camera_props">
+    <gazebo reference="${camera_link_name}">
+      <sensor name="${camera_link_name}_sensor" type="depth">
+        <visualize>true</visualize>
+        <update_rate>30.0</update_rate>
+        <camera name="camera">
+          <horizontal_fov>${camera_props['horizontal_fov']}</horizontal_fov>
+          <image>
+            <width>${camera_props['image']['width']}</width>
+            <height>${camera_props['image']['height']}</height>
+            <format>${camera_props['image']['format']}</format>
+          </image>
+          <clip>
+            <near>${camera_props['clip']['near']}</near>
+            <far>${camera_props['clip']['far']}</far>
+          </clip>
+        </camera>
+        <plugin name="depth_camera_controller" filename="libgazebo_ros_camera.so">
+          <cameraName>${camera_link_name}</cameraName>
+          <imageTopicName>${camera_link_name}_ir/image_raw</imageTopicName>
+          <cameraInfoTopicName>${camera_link_name}_ir/camera_info</cameraInfoTopicName>
+          <depthImageTopicName>${camera_link_name}_depth/image_raw</depthImageTopicName>
+          <depthImageCameraInfoTopicName>${camera_link_name}_depth/camera_info</depthImageCameraInfoTopicName>
+          <pointCloudTopicName>${camera_link_name}_depth/points</pointCloudTopicName>
+          <frameName>${camera_link_name}_optical_frame</frameName>
+
+          <baseline>0.2</baseline>
+          <alwaysOn>true</alwaysOn>
+          <updateRate>0.0</updateRate>
+          <frame_name>camera_depth_frame</frame_name>
+          <pointCloudCutoff>0.5</pointCloudCutoff>
+          <pointCloudCutoffMax>3.0</pointCloudCutoffMax>
+          <distortionK1>0</distortionK1>
+          <distortionK2>0</distortionK2>
+          <distortionK3>0</distortionK3>
+          <distortionT1>0</distortionT1>
+          <distortionT2>0</distortionT2>
+          <CxPrime>0</CxPrime>
+          <Cx>0</Cx>
+          <Cy>0</Cy>
+          <focalLength>0</focalLength>
+          <hackBaseline>0</hackBaseline>
```

Comment:
Some of these paraemeters might matter but we can keep them like this for now. Let's just keep it in mind that we kept a bunch of 0s for the intrinsic camera calibration matrix (including focal lengths). Don't need to do anything about this comment.